source/extensions/filters/http/wasm/wasm_filter.h

```diff
@@ -40,11 +40,11 @@ class FilterConfig : Logger::Loggable<Logger::Id::wasm> {
     }
     if (!wasm || wasm->isFailed()) {
       if (handle->plugin()->fail_open_) {
-        // Fail open skips adding this filter to callbacks.
-        return nullptr;
+        return nullptr; // Fail open skips adding this filter to callbacks.
+
```

Comment:
Nit: unnecessary empty line.

Reply:
Thanks! Interesting part is that this empty line is also counted as a uncovered line of codes.